packages/nextjs/pages/index.tsx

```diff
@@ -328,15 +328,16 @@ const Home: NextPage<{
 };
 
 export const getStaticProps: GetStaticProps<{ stats: Stats }> = async () => {
-  const res = await fetch(`${process.env.NEXT_PUBLIC_BG_BACKEND_API}/api/stats`);
-
-  if (!res.ok) throw new Error(`Failed to fetch stats, received status ${res.status}`);
-
-  const stats = (await res.json()) as Stats;
-
   return {
     props: {
-      stats,
+      stats: {
+        builderCount: "50",
+        buildCount: "100",
+        streamedEth: 100,
+        buildersIncrementMonth: 10,
+        buildsIncrementMonth: 10,
+        streamedEthIncrement: 10,
+      },
```

Comment:
I suppose this change was for your internal tests? We'd destroy the stats if we merge :)